src/ngCookies/cookieWriter.js

```diff
@@ -14,6 +14,11 @@
 function $$CookieWriter($document, $log, $browser) {
   var cookiePath = $browser.baseHref();
   var rawDocument = $document[0];
+  var sameSiteValidValues = ['lax', 'strict'];
```

Comment:
How big is the chance that another option is added to this? This validation would prevent us from supporting this seemlessly.